scripts/assume.bat

```diff
@@ -48,5 +48,32 @@ for /f "tokens=1,2,3,4,5,6,7 delims= " %%a in ("%ASSUME_OUTPUT%") do (
 
         Exit /b %ASSUME_STATUS%
     )
+
+    if "%%a" == "GrantedAssumeSSO" (
```

Comment:
@meyerjrr This looks like it would work, however the way the script is written, if you put the value "None" in place of the aws_profile output, it will not be set.